lib/cli/ui/prompt/interactive_options.rb

```diff
@@ -61,6 +61,7 @@ def initialize(options, multiple: false, default: nil)
           end
           @redraw = true
           @presented_options = []
+          @supports_arrow_keys = CLI::UI::OS.current.supports_arrow_keys?
```

Comment:
Maybe I'm not fully understand but this seems misleading? Windows _does_ support the arrow keys, it's just a different mechanism?

Reply:
I believe we need an external gem for that, windows doesn't capture the up/down arrow keys natively 
https://github.com/Shopify/cli-ui/pull/144

Reply:
Yes, if I'm not mistaken one can use the `curses` gem to properly capture events on Windows. One option would be to actually just import the actual gem code in the CLI repo, but that would involve some C code, so I'm not sure how easy that would be, and it would make the CLI a bit bigger.